.github/workflows/release.yml

```diff
@@ -20,9 +21,27 @@ jobs:
         with:
           name: hevm-x86_64-macos
           path: ./hevm-x86_64-macos
+  macosARMRelease:
+    name: Build MacOS Binary
+    needs: macosx86Release
+    # macos-14 is arm, see https://github.com/orgs/community/discussions/102846
+    #   "Workflows executed on this image will run exclusively on the 3 vCPU M1 runner"
+    runs-on: macos-14
+    steps:
+      - uses: actions/checkout@v4
+      - uses: DeterminateSystems/nix-installer-action@main
+      - uses: DeterminateSystems/magic-nix-cache-action@main
+      - name: build hevm
+        run: |
+          nix build .#redistributable --out-link hevmMacos
+          cp ./hevmMacos/bin/hevm ./hevm-arm64-macos
+      - uses: actions/upload-artifact@v4
+        with:
+          name: hevm-arm64-macos
+          path: ./hevm-arm64-macos
   linuxRelease:
     name: Create Release
-    needs: macosRelease
+    needs: macosARMRelease
```

Comment:
Out of curiosity, why do we need to sequentialize the jobs like this?
Why cannot we just remove the `needs` steps?

Reply:
No idea. I didn't want to mess with it, because it already was super-annoying and I spent 3-4 days chasing it. Let's see if it works without it...

Reply:
OK, I have now fixed this. Dependency removed.